docs/07-theme-developer-tutorial/02-remote-theme.md

```diff
@@ -0,0 +1,104 @@
+  );
+});
+```
+
+Now open up `common/common.scss`, and paste in:
+
+```css
+.welcome-banner {
+  background: green;
+  color: white;
+  text-align: center;
+  padding: 10px;
+}
+```
+
+Assuming `discourse_theme` is still running in your terminal, these changes will have been synced up to your Discourse site instantly, and reflected in the theme preview. Try tweaking some of the text in the files and watch the result change in the preview.
+
+To start up `discourse_theme`'s syncing again without going through the full 'new' flow, run `discourse_theme watch .` from inside your theme's directory. Add `--reset` if you need to change your forum URL or API key.
+
+## Understanding the file structure
+
+As you've probably realized, many of the files in the local directory are a 1:1 match to the tabs in the admin panel from chapter 1. For example, `common/common.scss` matches the CSS tab, `common/after_header.html` matches the After Header tab, and files under `javascripts/discourse/api-initializers/*.gjs` are each treated the same as the JS tab.
+
+Along with these, you may have noticed a bunch more placeholder directories & files which have been automatically generated by `discourse_theme`. We'll explore a few of these in more detail later in the tutorial, but you can always find a list of all available files [here](https://meta.discourse.org/t/60848).
+
+For now, take a look at the `about.json` file. This contains the most important metadata for your Remote Theme, including its name, description, author info, and various URLs. This information will be shown to any users of your theme, so it's important to keep it up-to-date.
+
+## Uploading to GitHub
+
+So far you've created a local directory and synced it up to a single forum via `discourse_theme`. To version-control your theme and make it available for others to install via the Discourse UI, you'll need to push the directory to an internet-accessible Git repository. We'll be using GitHub for this tutorial, but any git-hosting platform should work.
+
+First, let's prepare the local directory. Navigate to the theme in your terminal, then run
+
+```bash
+git add .
+git commit -m "My first commit"
+```
+
+Or if you prefer, perform the commit from your favorite Git user interface.
+
+Then set up the remote repository by visiting [github.com/new](https://github.com/new). Enter a name & description for your repository. Leave the template selection empty, and skip the README, gitignore and license options. Create the repository.
+
+Then follow GitHub's instructions to "push an existing repository from the command line". Once complete, you should see all your theme files in the GitHub UI. The repository URL can now be used to install the theme on any Discourse forum, and any updates you push to GitHub will be made available to forum admins.
+
+Now that we've got the mechanics out the way, we'll be exploring a bunch of ways that themes can be used to customize Discourse. Join us in [the next chapter].
```

Comment:
Link will be added in a followup PR, because we don't have a topic-id for this doc yet. (🐔 / 🥚 problem)